src/main/kotlin/com/mineinabyss/launchy/data/Config.kt

```diff
@@ -25,3 +30,41 @@ data class Config(
         }.getOrDefault(Config())
     }
 }
+
+@Throws(IOException::class)
+fun unzip(zipFilePath: File, destDirectory: String) {
+
+    File(destDirectory).run {
+        if (!exists()) {
+            mkdirs()
+        }
+    }
+
+    ZipFile(zipFilePath).use { zip ->
+        zip.entries().asSequence().forEach { entry ->
+            zip.getInputStream(entry).use { input ->
+                val filePath = destDirectory + File.separator + entry.name
+
+                if (!entry.isDirectory) extractFile(input, filePath)
+                else File(filePath).mkdir()
+            }
+        }
+    }
+}
+
+@Throws(IOException::class)
+fun extractFile(inputStream: InputStream, destFilePath: String) {
+    val bufferSize = 4096
+    val buffer = BufferedOutputStream(FileOutputStream(destFilePath))
+    val bytes = ByteArray(bufferSize)
+    var read: Int
+    while (inputStream.read(bytes).also { read = it } != -1) {
+        buffer.write(bytes, 0, read)
+    }
+    buffer.close()
+}
+
+suspend fun downloadConfig() {
+    val configUrl = "https://github.com/MineInAbyss/launchy-mods/releases/download/v1.0.0/configs.zip"
+    Downloader.download(configUrl, Dirs.mineinabyss / "configs.zip")
```

Comment:
Don't use a configs.zip string here, instead just add a directory variable to Dirs.